src/ObjectUtils/ElfFile.cpp

```diff
@@ -657,8 +666,6 @@ ErrorMessageOr<std::unique_ptr<ElfFile>> CreateElfFile(
 
 ErrorMessageOr<uint32_t> ElfFile::CalculateDebuglinkChecksum(
     const std::filesystem::path& file_path) {
-  // TODO(b/180995172): Make this read operation iterative since the potentially read files
-  // can be very large (gigabytes).
```

Comment:
This bug was closed as already implemented (the IDE was telling me the bug is closed).